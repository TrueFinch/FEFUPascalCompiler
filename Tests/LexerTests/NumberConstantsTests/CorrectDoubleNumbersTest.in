1.
123.12
0.12
1e1
1E1
1e-1
1E-1
1.e1
1.e-1
1.E1
1.E-1
1.12e1
1.12e-1
1.12E1
1.12E-1