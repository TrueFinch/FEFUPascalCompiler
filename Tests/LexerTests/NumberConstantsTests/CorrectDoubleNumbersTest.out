(1, 1)    | FloatNumber         | 1.                            | 1                             |
(2, 1)    | FloatNumber         | 123.12                        | 123,12                        |
(3, 1)    | FloatNumber         | 0.12                          | 0,12                          |
(4, 1)    | FloatNumber         | 1e1                           | 10                            |
(5, 1)    | FloatNumber         | 1E1                           | 10                            |
(6, 1)    | FloatNumber         | 1e-1                          | 0,1                           |
(7, 1)    | FloatNumber         | 1E-1                          | 0,1                           |
(8, 1)    | FloatNumber         | 1.e1                          | 10                            |
(9, 1)    | FloatNumber         | 1.e-1                         | 0,1                           |
(10, 1)   | FloatNumber         | 1.E1                          | 10                            |
(11, 1)   | FloatNumber         | 1.E-1                         | 0,1                           |
(12, 1)   | FloatNumber         | 1.12e1                        | 11,2                          |
(13, 1)   | FloatNumber         | 1.12e-1                       | 0,112                         |
(14, 1)   | FloatNumber         | 1.12E1                        | 11,2                          |
(15, 1)   | FloatNumber         | 1.12E-1                       | 0,112                         |